src/qml/MessagesListView.qml

```diff
@@ -87,7 +87,6 @@ MultipleSelectionListView {
         id: loader
         anchors.left: parent.left
         anchors.right: parent.right
-        height: status == Loader.Ready ? item.height : 0
```

Comment:
Why was this here in the first place?

Reply:
Probably to prevent errors when the Loader hadn't completed loading yet: https://github.com/ubports/messaging-app/commit/91f652c8922a572e7e72d9b1e1056582a21bb092

Reply:
No idea. Probably was working around some issue in older Qt, which is now resolved with the improvements to the QtQuick engine.

Reply:
don't see any regressions, i've tested it already before

Reply:
A good test result is a resolution to me.